src/js/browser-plugin.js

```diff
@@ -1220,7 +1213,7 @@ function openLabelPopup(tweet) {
     confirmButtonText: buttonText,
     reverseButtons: true,
     focusCancel: true,
-    onClose: sendCloseLabelPopUpLog(node),
+    onClose: sendCloseLabelPopUpLog,
```

Comment:
If we do like this, then when the "sendCloseLabelPopUpLog" function will be executed, the parameters will be empty. I think it is better to just copy the function code here: like, for example, how it is done in the "preConform" action in the popup opened on line 1353.